metricflow/test/fixtures/id_fixtures.py

```diff
@@ -19,7 +19,7 @@ class IdNumberSpace:
     CONSISTENT_ID_REPOSITORY = 10000
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(autouse=True, scope="function")
```

Comment:
Huh, the function-scope here may be what imposes id number consistency, but this shouldn't be a change in behavior since scope defaults to function so I'm not really sure what you were running into or what this fixes.

Reply:
This was left over from some testing iteration - I'll remove.